testfiles/1ZoneUncontrolled3SurfaceZone.idf

```diff
@@ -0,0 +1,397 @@
+!-Generator IDFEditor 1.34
+!-Option OriginalOrderTop UseSpecialFormat
+!-NOTE: All comments with '!-' are ignored by the IDFEditor and are generated automatically.
+!-      Use '!' comments if they need to be retained when using the IDFEditor.
+!1ZoneUncontrolled3SurfaceZone.idf
```

Comment:
@RKStrand Is the intent to add this file to the internal test suite?  If so, it needs a line in testfiles/CMakeLists.txt  Also, this file has warmup convergence errors, so it appears that something isn't working right with 3 surfaces.